File: Hello.java
//Hello.java

import java.util.*;

public class Hello {
    public static void main (String[] args){
      Scanner input = new Scanner(System.in);
      String userName;
      System.out.println("What is your name?");
      userName = input.nextLine();
      System.out.println("Hi there, " + userName + "!");
    } // end main
} 

File: hello.c
// be sure we can do input and output
#include <stdio.h>

int main(){
  // set aside memory called userName
  char userName[20];
  //Ask user name
  printf("What is your name? ");
  //put input into userName
  scanf("%s", userName);
  //print greeting incorporating userName
  printf("Hi there, %s! \n", userName);
  return(0);
} // end main

